core/src/modules/ops/record.ts

```diff
@@ -945,38 +960,112 @@ export namespace RecordOps {
     limit = 100,
     toExport = true
   ) {
-    let records: GrouparooRecord[] = await api.sequelize.query(
+    const records: GrouparooRecord[] = await api.sequelize.query(
       `
-    SELECT "id" from "records" where "state" = 'pending'
-    EXCEPT
-    SELECT DISTINCT("recordId") FROM "recordProperties" WHERE "state" = 'pending'
-    LIMIT ${limit}
-    ;
+      SELECT "id" from "records" where "state" = 'pending'
+      EXCEPT
+      SELECT DISTINCT("recordId") FROM "recordProperties" WHERE "state" = 'pending'
+      LIMIT ${limit};
     `,
       {
         type: QueryTypes.SELECT,
         model: GrouparooRecord,
+        include: RecordProperty,
```

Comment:
whoa! I wouldn't have expected this to work since you aren't joining the record properties here.  How does this modify the SQL generated?